### ISSUE TITLE
Consistency parameter in independence_test_method

### ISSUE THREAD

**Issue by priamai** [py-why/causal-learn#151]
I noticed that for example the CDNOD calls them: 
indep_test: Independence test method function. Default: ‘fisherz’.
But FCI:
independence_test_method: Independence test method function. Default: ‘fisherz’.

I think we should unify all the methods to use the same keyword?
Cheers.

**Comment by kunwuz** [comment]
Yeah, this is definitely something on the list. We will improve the APIs shortly. Thanks for your suggestion!
